src/Components/test/testassets/TestServer/Program.cs

```diff
@@ -23,6 +23,7 @@ public static void Main(string[] args)
                 ["Server authentication"] = (BuildWebHost<ServerAuthenticationStartup>(CreateAdditionalArgs(args)), "/subdir"),
                 ["CORS (WASM)"] = (BuildWebHost<CorsStartup>(CreateAdditionalArgs(args)), "/subdir"),
                 ["Prerendering (Server-side)"] = (BuildWebHost<PrerenderedStartup>(CreateAdditionalArgs(args)), "/prerendered"),
+                ["Hosted in AspNet"] = (BuildWebHost<HostedInAspNetStartup>(CreateAdditionalArgs(args)), "/subdir"),
```

Comment:
As above, not sure why this is needed and would ideally prefer not to expand this set if we can avoid it. Hope that’s ok!

Reply:
Duplicate test app has been removed now.